### PR TITLE
example's source code updates with additional edits for improving user experience.

### PR DESCRIPTION
**Pull request by higgo3636**
#### What's this PR do?

Provides some needed source code snippets for component examples on some of the pages and other edits to improve usability.

#### Where should the reviewer start?

Can check each page's source code examples updated and make sure everything looks correct.  Also should see some additional styling added to help differ from rest of the the page and some script updates to the index.js file to help with the tours + filter page demo's functionality.

#### How should this be manually tested?

Reference notes directly above.

#### Any background context you want to provide?

The team wanted to further improve upon user experience.

#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
- Is there a blog post?
n/a
- Does the knowledge base need an update?
n/a
- Does this add new (Python) dependencies which need to be added to chef?
n/a

### CODE REVIEW

File: src/front/styles/pattern-library.scss [hunk 1]
```diff
@@ -564,3 +564,24 @@ hr.hr-strong {
...
+}
+
+.pl-pattern {
+    #myModalLive .modal-dialog {
+        margin: 30% auto;
+        max-width: 90%; 
+    }
+}
+
+.filter-spacer {
```
Comment (davezen1) [inline src/front/styles/pattern-library.scss]: Maybe we prefix pl to this to avoid confusion?

Comment (higgo3636) [reply src/front/styles/pattern-library.scss]: good idea, this is done.

Comment (higgo3636) [reply src/front/styles/pattern-library.scss]: ok, all requested updates should be done.

File: src/docs/patterns/wizards.md [hunk 2]
```diff
@@ -67,38 +68,95 @@ Use vertical steps when there are 5 or more steps, when there are nested steps,
...
-                          <label for="ht4" class="mdl-textfield__label">Email</label>
-                          <input class="mdl-textfield__input" type="text" pattern="^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$" id="ht4">
-                          <p class="mdl-textfield__help">Email address of alert recipient</p>
-                          <span class="mdl-textfield__error"><i class="fa fa-exclamation-triangle fa-fw"></i>&nbsp;Not a valid email address</span>
-                        </div>
-                      </div>
-                      <button type="button" class="btn btn-default">Cancel</button>
-                      <button type="submit" class="btn btn-primary pull-right">Continue</button>
-                    </form>
+<form class="form" style="max-width: 450px;" role="form">
```
Comment (davezen1) [inline src/docs/patterns/wizards.md]: We lost indentation here as well

File: src/front/scripts/index.js [hunk 3]
```diff
@@ -352,46 +352,76 @@ $(document).ready(function () {
...
-      showPrevButton: true
-  };
-  var calloutMgr = hopscotch.getCalloutManager();
-  
-  if (document.URL.indexOf('tours.html') > -1) {
-      // Start the tour!
-      hopscotch.startTour(tour);
-
-      calloutMgr.createCallout({
+     // Hopscotch
```
Comment (davezen1) [inline src/front/scripts/index.js]: On load the Callout appears in the wrong place:

tours

**Pull request merged**